editor/src/components/canvas/canvas-strategies/strategies/group-like-helpers.ts

```diff
@@ -162,3 +167,33 @@ export function treatElementAsContentAffecting(
 ): boolean {
   return getElementContentAffectingType(metadata, allElementProps, path) != null
 }
+
+export const GroupFlagKey = 'data-group'
+
+export function isGroupPragmaApplied(
```

Comment:
I think pragma has a ton of baggage, I'd pick a name which is a little more on the safe side about using big words  

```suggestion
export function isElementMarkedAsGroup(
```

Reply:
makes perfect sense, renamed in https://github.com/concrete-utopia/utopia/pull/3504/commits/c79e4f708be85ebe7f46c7c022cd725b33eb186e